packages/core/src/Lib/UseTailwindAnimation/Internal/Lib/UseTailwindAnimation.ts

```
@@ -0,0 +1,17 @@
+import type { TailwindAnimation }             from '@/Lib/UseTailwindAnimation/Types';
+import type { ComputedRef, MaybeRefOrGetter } from 'vue';
```

ESLint

[error] Expected "vue" (type) to come before "@/Lib/UseTailwindAnimation/Types" (internal-type).

(perfectionist/sort-imports)